lib/svg/__init__.py

```diff
@@ -1,4 +1,4 @@
+from .guides import get_guides
+from .path import apply_transforms, get_node_transform, get_correction_transform, line_strings_to_csp, point_lists_to_csp, line_strings_to_path
```

Comment:
Do you lint your inits or imports? I would make this newline two lines

Reply:
I do!  See the `style` target in the Makefile.  I chose to set the maximum line length to 150, since 80 columns seems too restrictive given today's monitor resolutions.

FWIW, I use Eclipse with PyDev and have it set to run `autopep8` whenever I save.  It sorts my imports and splits long lines automatically.